server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -174,6 +181,42 @@ public SchedulerStatus setSchedulerStatus(boolean running) {
         return getSchedulerStatus();
     }
 
+    @POST
+    @Path("cron/{task}")
```

Comment:
I'm not even sure I understand the difference without sifting through each job and looking for DEFAULT_SCHEDULE.